addons/website_event/security/ir.model.access.csv

```diff
@@ -1,11 +1,8 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-access_event_event_public,event.event.public,event.model_event_event,base.group_public,1,0,0,0
-access_event_event_portal,event.event.portal,event.model_event_event,base.group_portal,1,0,0,0
 access_event_type,event.type,event.model_event_type,,0,0,0,0
-access_event_event_ticket_public,event.event.ticket.public,event.model_event_event_ticket,base.group_public,1,0,0,0
-access_event_event_ticket_portal,event.event.ticket.portal,event.model_event_event_ticket,base.group_portal,1,0,0,0
 access_event_tag_category,event.tag.category,event.model_event_tag_category,,1,0,0,0
 access_event_tag,event.tag,event.model_event_tag,,1,0,0,0
 access_website_event_menu,website.event.menu,model_website_event_menu,,1,0,0,0
-access_website_event_menu_manager,website.event.menu.manager,model_website_event_menu,event.group_event_manager,1,1,1,1
+access_website_event_menu_user,website.event.menu.user,model_website_event_menu,event.group_event_user,1,1,1,1
 access_website_visitor_user,website.visitor.user,model_website_visitor,event.group_event_user,1,1,0,0
+access_event_event,event.event,model_event_event,,1,0,0,0
```

Comment:
Maybe put that line at the beginning of the file since it impacts the "base" model?

Otherwise LGTM.
We will see what the reviewer thinks about the addition of archivable models since I'm still not sold on that solution.